src/state/apis/swapper/helpers/validateTradeQuote.ts

```diff
@@ -36,12 +36,16 @@ export const validateTradeQuote = async (
     error,
     isTradingActiveOnSellPool,
     isTradingActiveOnBuyPool,
+    sendAddress,
   }: {
     swapperName: SwapperName
     quote: TradeQuote | undefined
     error: SwapErrorRight | undefined
     isTradingActiveOnSellPool: boolean
     isTradingActiveOnBuyPool: boolean
+    // TODO(gomes): this should most likely live in the quote alongside the receiveAddress,
```

Comment:
Agreed

Reply:
@woodenfurniture when you're back for vis ^